config/fortunes.json

```diff
@@ -1,338 +1,242 @@
 [
   {
     "id": 0,
```

Comment:
Ids are used in localStorage to remember which cookies were shown. So, we mustn't changed text in the existing cookies.
Please, add 100 to all ids in order to prevent problems with cookies displaying.
@fairlighteth this is important

Reply:
See `checkedFortunesListAtom` for more details

Reply:
Thanks for checking this. Pushed an update to the numbering.